assembly/__tests__/runtime/main.ts

```diff
@@ -557,3 +557,10 @@ export function setValue(value: string): string {
 export function getValue(): string | null {
   return storage.get<string>("name");
 }
+
+export function getDefaultValue(y: string = "hello world", x: i32 = 42): i32 {
+  assert(x == 42, "x is not default value of 42");
+  assert(y == "hello world", "y is not default value of hello world");
```

Comment:
Hello world is not default value of y?

Reply:
It is.  This is a test that when compiled and the default values still work when the function is called in the nearcore test.